
Author: rmh
Description:  Move _PATH_LOG to paths.h from sysdeps (this way we can override it)
Status: not sent

diff -ur libc.old/misc/sys/syslog.h libc/misc/sys/syslog.h
--- libc.old/misc/sys/syslog.h	2006-01-14 13:09:36.000000000 +0100
+++ libc/misc/sys/syslog.h	2006-01-17 11:29:03.000000000 +0100
@@ -35,9 +35,7 @@
 #include <features.h>
 #define __need___va_list
 #include <stdarg.h>
-
-
-#define	_PATH_LOG	"/dev/log"
+#include <paths.h>
 
 /*
  * priorities/facilities are encoded into a single 32-bit quantity, where the
diff -ur libc.old/sysdeps/generic/paths.h libc/sysdeps/generic/paths.h
--- libc.old/sysdeps/generic/paths.h	1999-06-08 23:04:36.000000000 +0200
+++ libc/sysdeps/generic/paths.h	2006-01-17 11:33:43.000000000 +0100
@@ -47,6 +47,7 @@
 #define	_PATH_KMEM	"/dev/kmem"
 #define	_PATH_MAILDIR	"/var/mail"
 #define	_PATH_LASTLOG	"/var/log/lastlog"
+#define	_PATH_LOG	"/dev/log"
 #define	_PATH_MAN	"/usr/man"
 #define	_PATH_MEM	"/dev/mem"
 #define	_PATH_MNTTAB    "/etc/fstab"
diff -ur libc.old/sysdeps/unix/sysv/linux/paths.h libc/sysdeps/unix/sysv/linux/paths.h
--- libc.old/sysdeps/unix/sysv/linux/paths.h	2000-11-16 00:06:47.000000000 +0100
+++ libc/sysdeps/unix/sysv/linux/paths.h	2006-01-17 11:33:58.000000000 +0100
@@ -47,6 +47,7 @@
 #define	_PATH_KLOG	"/proc/kmsg"
 #define	_PATH_KMEM	"/dev/kmem"
 #define	_PATH_LASTLOG	"/var/log/lastlog"
+#define	_PATH_LOG	"/dev/log"
 #define	_PATH_MAILDIR	"/var/mail"
 #define	_PATH_MAN	"/usr/share/man"
 #define	_PATH_MEM	"/dev/mem"
